src/geometry.rs

```diff
@@ -70,6 +73,7 @@ pub struct Geometry {
     pub shapes: HashMap<String, Shape>,
     /// Faces.
     pub faces: Vec<[u32; 3]>,
+    #[doc(hidden)] pub __id: usize,
```

Comment:
does it have to be public?

Reply:
Unfortunately :( You can't use nice `.. Default::default() ` syntax on structs with private fields